simpeg/optimization.py

```diff
@@ -1055,13 +1188,14 @@ class InexactGaussNewton(BFGS, Minimize, Remember):
 
     """
 
-    def __init__(self, **kwargs):
-        Minimize.__init__(self, **kwargs)
+    def __init__(self, *, cg_rtol=1e-1, cg_atol=0.0, cg_maxiter=5, **kwargs):
+        super().__init__(
+            cg_rtol=cg_rtol, cg_atol=cg_atol, cg_maxiter=cg_maxiter, **kwargs
+        )
 
-    name = "Inexact Gauss Newton"
+        self._was_default_hinv = False
 
-    maxIterCG = 5
-    tolCG = 1e-1
+    name = "Inexact Gauss Newton"
 
     @property
     def approxHinv(self):
```

Comment:
This should likely be moved to the `InexactCG` class and then used within all of the `Inexact` step direction minimizers. It's left here due to the default BFGS preconditioner.